### PR TITLE
Rewrite installation instructions

### PR DESCRIPTION
**Pull request by kxmo**
I've recently setup Loomio using the old instructions and saw room for some improvements:
- Rearranged some of the instructions - general instructions in setup_environment.md and Loomio specific in quicksetup.
- Added some instructions that were missing
- Removed unneeded steps or those that could become out of date
I have tested these instructions on Ubuntu 14.04. I don't have access to a Mac but it would be good if someone could test them there.

### CODE REVIEW

File: docs/en/development_handbook/quickstart.md [hunk 1]
```diff
@@ -33,16 +64,22 @@ _NB: Take note of the email and password generated at the end of this task; you'
 
 ## Build the frontend client
 ```
-cd client && gulp compile
+gulp compile
 ```
 ## Launch the server
 ```
+cd angular
```
Comment (robguthrie) [inline docs/en/development_handbook/quickstart.md]: Are you sure you intended to write angular? I think you want to cd to client here.

**Comment by robguthrie** [comment]
Thanks for this! Looks pretty good.

File: docs/en/development_handbook/quickstart.md (hunk 1 above)
Comment (kxmo) [reply docs/en/development_handbook/quickstart.md]: If the user keeps the same terminal instance open they're already in client (line 54).
I think gulp changes the working directory anyway so this step may not be needed

**Comment by gdpelican** [comment]
This is amazing, thanks @kxmo !

**Pull request merged**